crates/sui-replay/src/lib.rs

```diff
@@ -169,6 +172,9 @@ pub async fn execute_replay_command(
             if show_effects {
                 println!("{:#?}", sandbox_state.local_exec_effects);
             }
+            if diag {
+                println!("{:#?}", sandbox_state.pre_exec_diag);
```

Comment:
nit: Seeing as this is the pre execution diagnostics, can we print it before the effects?

Reply:
Sure